framework/src/postprocessors/NumRelationshipManagers.C

```diff
@@ -46,19 +46,19 @@ NumRelationshipManagers::getValue()
   unsigned int count = 0;
   if (_rm_type == "GEOMETRIC")
   {
-    for (const auto rm : rms)
+    for (const auto & rm : rms)
```

Comment:
`rm` is a pointer, so let's instead do `&` -> `* const`

Reply:
Oops - I got lazy and followed the compiler's suggestion. 😅 Thanks!

Reply:
Actually, clang didn't like that extra `const`, since the type ends up as `const std::shared_ptr<RelationshipManager>`, so I'll stick with just a `*`.

Reply:
Clang isn't happy with either `*` or `* const`:

```
/Users/icenct/projects/moose/framework/src/postprocessors/NumRelationshipManagers.C:49:23: error: variable 'rm' with type 'const auto *' has incompatible initializer of type 'const std::__1::shared_ptr<RelationshipManager>'
    for (const auto * rm : rms)
                      ^  ~
```

Shouldn't this declared variable be a reference to that shared pointer?

Reply:
Then I've led you astray. Since it's a `shared_ptr`, then the `&`s are good